lib/simplecov.rb

```diff
@@ -49,11 +49,44 @@ def start(profile = nil, &block)
     end
 
     #
+    # Finds files that were to be tracked but were not covered and initializes
+    # their coverage to zero, with an estimation of the line count.
+    #
+    def add_uncovered_files(result)
+      if @track_files_glob
```

Comment:
I think it's reasonable to say that if you are telling SimpleCov you expect a file to be used, but it isn't even loaded, to count that as zero.

However, I understand your concern that any Ruby file that is loaded has a non zero 'coverage', so perhaps, if the file isn't loaded at the time to results are calculated, we just load the file and generate the coverage for those, as usual?